src/modules/dashboard/components/ColonyHomeActions/ColonyHomeActions.tsx

```diff
@@ -103,6 +112,13 @@ const ColonyHomeActions = ({ colony }: Props) => {
         colony,
       },
     },
+    {
+      component: EditDomainDialog,
+      props: {
+        prevStep: 'dashboard.ManageDomainsDialog',
+        colony,
+      },
+    },
```

Comment:
Is this something from the rebase ?

Reply:
what you mean here?

Reply:
I mean this PR doesn't have anything to do with the Edit Domain functionality.

So I found it weird that there's a change regarding it.

Reply:
i believe it can be just order change